spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -1128,4 +1165,48 @@ private boolean determineSslEnabled(boolean sslEnabled) {
 
 	}
 
+	public static final class Stream {
+
+		private String host = "localhost";
+
+		private int port = DEFAULT_STREAM_PORT;
```

Comment:
This property needs some javadoc that'll be captured in the configuration property metadata.